command.c

```diff
@@ -1230,8 +1232,9 @@ void redis_parse_cmd(struct cmd *r) {
             break;
 
         case SW_ARG1:
-            m = p + rlen;
+            m = p + rlen; // Move forward given length ($ in protocol)
             if (m >= cmd_end) {
+                // Moving past the end, not good..
```

Comment:
These comments don't say more than the actual code, given that you know what the variable names mean. If the variable names are documented where they are declared (or even changed into better variable names) I guess these comments won't be needed. What do you think?

Reply:
Will remove